src/components/HomeCards/EQPProjectsCard/EQPProjectsCard.module.css

```diff
@@ -44,6 +48,7 @@
     font-weight: 400;
     line-height: 24px;
     color: var(--gray-8);
+    flex: 1;
```

Comment:
pas necessaire

Reply:
this is required to be added to the text. If not, the contents will not be alligned...
![Screenshot from 2025-04-01 13-29-56](https://github.com/user-attachments/assets/fd86b914-fc75-4ef4-96e5-883d875838a9)